pkg/network/user_proxy.go

```diff
@@ -55,3 +61,34 @@ func (p *Proxy) Create(user entities.User) (err error) {
 
 	return nil
 }
+
+// CreateToken genereates a token for the specified user
+func (p *Proxy) CreateToken(user entities.User) (string, error) {
+	var resp *http.Response
+
+	parsedCredentials, err := json.Marshal(user)
+	if err != nil {
+		return "", err
+	}
+
+	client := &http.Client{}
+	resp, err = client.Post(p.url+"/tokens", "application/json", bytes.NewBuffer(parsedCredentials))
+	if err != nil {
+		return "", err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode == http.StatusForbidden {
+		msg := fmt.Sprintf("Invalid credentials %s", user)
```

Comment:
Avoid printing the struct user since it can display the data on the message response. The password in plain text will be displayed too.

Reply:
Done.